assets/dev/js/editor/container/container.js

```diff
@@ -419,6 +470,30 @@ export default class Container extends ArgsObject {
 		return false;
 	}
 
+	/**
+	 * Function forSomeChildrenRecursive().
+	 *
+	 * Will run over children recursively, breaks if the callback return true.
+	 *
+	 * @param {function(container:Container)} callback
+	 *
+	 */
+	forSomeChildrenRecursive( callback ) {
+		if ( callback( this ) ) {
+			return true;
```

Comment:
What if it has children? (e.g. nested)

Reply:
Handled in test.

Reply:
It should stop when callback is positive, if it's nested there are the rest of the method to handle it